app/components/autocomplete/index.js

```diff
@@ -36,6 +37,7 @@ export default class Autocomplete extends Component {
             <View>
```

Comment:
This solves the issue of having to wrap the ```<Autocomplete/>``` in a view in the ```<PostTextbox/>```. If it wasn't here then it would be wrapped in the ```<PostTextbox/>```. We have to wrap the absolute positioned component with a regular view to avoid the following: 
![image](https://cloud.githubusercontent.com/assets/5090577/23137579/56907114-f768-11e6-896b-55a27f7857ce.png)